### PR TITLE
Add Building docs

### PR DESCRIPTION
**Pull request by OlekAleksander**

### CODE REVIEW

File: _sidebar.md [hunk 1]
```diff
@@ -12,10 +12,13 @@
         * [NGINX without SSL](latest/panel/webservers/nginx.md)
         * [Apache with SSL](latest/panel/webservers/apache-ssl.md)
         * [Apache without SSL](latest/panel/webservers/apache.md)
-
+    * `Building panel assets`
```
Comment (cmrxnn) [inline _sidebar.md:15]: Just change this to 'Modification'

File: latest/panel/building/build.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Building panel assets
+
+!> Please note that we dont provide support to modified installs
```
Comment (cmrxnn) [inline latest/panel/building/build.md:3]: !> Please note that support can't be provided in the Jexactyl Discord if you choose to modify the Panel. You are doing this at your own risk.

**Comment by camwhi-te** [comment]
The formatting of these pages are incorrect with some typos, I've decided it'd be easier if I went ahead and resolved these myself and pushed them. Thanks for the suggestion.

**Comment by OlekAleksander** [comment]
okay

**Pull request closed without merging**